docs/python/python-tutorial.md

```diff
@@ -252,36 +253,33 @@ A best practice among Python developers is to avoid installing packages into a g
 
 > **Note**: For additional information about virtual environments, see [Environments](/docs/python/environments.md#creating-environments).
 
-1. Create and activate the virtual environment
+1. Create a Virtual Environment using the Create Environment Command
 
-   **Virtual environment creation for Windows**
+   From within VS Code, you can create non-global environments, using Venv or Anaconda, by opening the Command Palette (Ctrl+Shift+P), start typing the `Python: Create Environment` command to search, and then select the command. You can also trigger the `Python: Create Environment' command through the Getting Started with Python page.
 
-   ```bat
-   py -3 -m venv .venv
-   .venv\scripts\activate
-   ```
+   The command presents a list of environment types, Venv or Conda.
 
-   If the activate command generates the message "Activate.ps1 is not digitally signed. You cannot run this script on the
-   current system.", then you need to temporarily change the PowerShell execution policy to allow scripts to
-   run (see [About Execution Policies](https://go.microsoft.com/fwlink/?LinkID=135170) in the PowerShell documentation):
+   ![Create Environment dropdown](images/environments/create_environment_dropdown.png)
 
-   ```powershell
-   Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process
-   ```
+   If you are creating an environment using **Venv**, the command presents a list of interpreters that can be used for your project.
 
-   **Virtual environment creation for macOS/Linux**
+   ![Virtual environment interpreter selection](images/environments/interpreters-list.png)
 
-   ```bash
-   python3 -m venv .venv
-   source .venv/bin/activate
-   ```
-   >**Note**: When you create a new virtual environment, you should be prompted by VS Code to set it as the default for your workspace folder. If selected, the environment will automatically be activated when you open a new terminal.
+   If you are creating an environment using **Conda**, the command presents a list of Python versions that can be used for your project.
+
+   ![Conda environment Python version selection](images/environments/conda_environment_python_versions.png)
+
+   After selecting the desired interpreter or Python version, a notification will show the progress of the environment creation and the environment folder will appear in your workspace.
+
+   ![Create environment status notification](images/environments/create_environment_prompt_status.png)
+
+   The command will also install necessary packages outlined in a requirements/dependencies file, such as requirements.txt, pyproject.toml or environment.yml, located in the   project folder.
```

Comment:
Please use inline code blocks for filenames so it would be:
"such as `requirements.txt`, `pyproject.toml`, or `environment.yml`"

Reply:
There is also an extra space before "project folder"

Reply:
fixed